src/app/containers/CpsRelatedContent/index.jsx

```diff
@@ -63,6 +77,8 @@ const CpsRelatedContent = ({ content, enableGridWrapper }) => {
     children: node.isRequired,
   };
   if (!content.length) return null;
+  const hasSingleRelatedContent = content.filter(Boolean).length === 1;
```

Comment:
What is the use-case for using `content.filter(Boolean).length === 1` could we just have a conditional in the render with something like `if(content.length > 1)`

Reply:
The use-case was in case the data somehow, is `[null, null]`, but I guess the propTypes takes care of that.